bin/generate-html.js

```diff
@@ -0,0 +1,33 @@
+#! /usr/bin/env node
+"use strict";
+const defaults = {
+  baseUrl: "./"
+};
+
+function template(rawOptions) {
+  const options = Object.assign({}, defaults, rawOptions || {});
+  return `
+<!doctype html>
+<html lang=en-us>
+  <head>
+    <meta charset="utf8">
```

Comment:
> Error: Bad value `utf8` for attribute `charset` on element meta: `utf8` is not a preferred encoding name. The preferred label for this encoding is `utf-8`.
> &mdash; via https://html5.validator.nu/

Also, not sure if we want to add a dummy `<title>` tag:

> Error: Element `head` is missing a required instance of child element `title`.
From line 7, column 3; to line 7, column 9